tutorials/ar-subs/index.md

```diff
@@ -0,0 +1,795 @@
+        card 1: Mic [Samson Meteor Mic], device 0: USB Audio [USB Audio]
+          Subdevices: 1/1
+          Subdevice #0: subdevice #0
+
+1. To identify the **sound input** options, execute `arecord -l`. In this example
+the USB microphone is card: `1`, device: `0`:
+
+        **** List of CAPTURE Hardware Devices ****
+        card 1: Mic [Samson Meteor Mic], device 0: USB Audio [USB Audio]
+          Subdevices: 1/1
+          Subdevice #0: subdevice #0
+
+1. To configure the OS to use the correct sound playback and microphone devices,
+edit or create the configuration file: `/home/pi/.asoundrc` (Note the dot in the
+  file name `.asoundrc`. It is a so-called hidden configuration file.). Add
+  the following content to the file and set the "mic" and "speaker" device
+  numbers to be the same as your `aplay -l` and `arecord -l` output findings.
+  In the following example, the microphone is set to "hw:1,0" which means
+  card: 1 and device: 0 which maps to the USB microphone. And the "speaker" is
+  set to card: 0 and device: 0 which maps to the Raspberry Pi built-in sound
+  card's 3.5mm audio output.
+
+        pcm.!default {
+          type asym
+          capture.pcm "mic"
+          playback.pcm "speaker"
+        }
+        pcm.mic {
+          type plug
+          slave {
+            pcm "hw:1,0"
+          }
+        }
+        pcm.speaker {
+          type plug
+          slave {
+            pcm "hw:0,0"
+          }
+        }
+
+
+### Test recording with the microphone
+
+1.  To test your microphone configuration, you can use the following command to
+record a 5 second clip in 16KHz raw format:
+
+        arecord --format=S16_LE --duration=5 --rate=16000 --file-type=raw out.raw
+
+1.  To check the audio playback device and the recorded audio file, execute:
+
+        aplay --format=S16_LE --rate=16000 out.raw
+
+**Note:** you have to connect speakers or headphones to the configured sound
+output device, such as the Raspberry Pi 3.5mm audio output.
+
+
+## Clone the example app and install its dependencies
+
+1.  Using the SSH or console connection to the Pi, clone the repository
+associated with the Google Cloud community tutorials:
+
+        git clone https://github.com/GoogleCloudPlatform/community.git
+
+        NOTE! Googlers, use this during the review stage:
+        gcloud source repos clone ar-subs --project=solution-demo
+
+1.  Change to the tutorial directory:
+
+        community/tutorials/ar-subs
+
+1.  Create a Python 3 virtual environment:
+
+        python3 -m venv venv
+
+1.  Activate the virtualenv:
+
+        source venv/bin/activate
+
+1.  Upgrade pip and setuptools:
+
+        pip3 install -U pip setuptools
+
+1.  Install the required Python modules:
+
+        pip3 install -r requirements.txt
+
+
+### Create a Service Account and JSON key
+
+Next we will create a Service Account in your GCP project, and grant sufficient
+permissions to it, so that it can use the AI services. We will also need to
```

Comment:
Use 'you' instead of 'we' to address the reader, as you do in the rest of the tutorial

Reply:
Done